projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -121,7 +121,12 @@ control_service_test_helm_chart:
       - projects/control-service/projects/helm_charts/pipelines-control-service/version.txt
 
 control_service_publish_job_base_image:
-  extends: .images:dind
+  image: docker:19.03.15
```

Comment:
20.10.23 (in favour of [20.10.22](https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/cicd/.gitlab-ci.yml#L8)) does not solve the issue? what was the issue?

Reply:
Just reverted the version in order to unblock the pipeline - more details here - https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3666206273.